pkg/etwlogrus/opts.go

```diff
@@ -44,7 +44,7 @@ func WithGetName(f func(*logrus.Entry) string) HookOpt {
 	}
 }
 
-// WithAdditionalEventOpts allows additional ETW event properties (keywords, tags, etc.) to be specified
+// WithAdditionalEventOpts allows additional ETW event properties (keywords, tags, etc.) to be specified.
```

Comment:
Looks like the name of the func changed at some point 😄

Reply:
i totally missed that 😅